src/benchmark.cc

```diff
@@ -81,6 +81,18 @@ BM_DEFINE_string(benchmark_filter, "");
 // benchmark execution, regardless of number of threads.
 BM_DEFINE_string(benchmark_min_time, kDefaultMinTimeStr);
 
+// Specification of required relative accuracy used to determine how
+// long to run the benchmark.
+//
+// REQUIRES: time is measured manually.
+//
+// Manual timers provide per-iteration times. The relative accuracy is
+// measured as the standard deviation of these per-iteration times divided by
+// the mean and the square root of the number of iterations. The benchmark is
+// run until the specified minimum time or number of iterations is reached
+// and the measured relative accuracy meets the specified requirement.
+BM_DEFINE_double(benchmark_min_rel_accuracy, 0.0);
```

Comment:
```suggestion
// Manual timers provide per-iteration times. The relative accuracy is
// measured as the standard deviation of these per-iteration times divided by
// the mean and the square root of the number of iterations. The benchmark is
// run until the following conditions are fulfilled:
//   1. the specified minimum time or number of iterations is reached
//   2. the measured relative accuracy meets the specified requirement
BM_DEFINE_double(benchmark_min_rel_accuracy, 0.0);
```
(I find your formulation prone to misunderstandings)

Reply:
Thanks and done. Better indeed.